README.md

```diff
@@ -28,7 +28,7 @@ The educandu framework
  | sessionDurationInMinutes | The validity of the user session in minutes | `number`, minumum 1 | no, defaults to 60 |
  | smtpOptions | The SMTP setup for sending emails to users upon registration or password reset | anything | yes |
  | emailSenderAddress | The email address from which emails are sent | `string` | yes |
- | bundleConfig | The same object that is also used to hydrate the app on the client side | `{ getPageTemplateComponent }` | yes |
+ | bundleConfig | The same object that is also used to hydrate the app on the client side | `{ getPageTemplateComponent, getSiteLogoComponent }` | yes, accepts `null` for either property and it will default to the internal setup
```

Comment:
Nice. Although it is only needed for the index page, and this gets injected into all pages. But I hope we will have the time during the next months to create a proper concept for templating (also on an individual page level).